slack_bolt/app/app.py

```diff
@@ -799,10 +799,20 @@ def _register_listener(
 # -------------------------
 
 
-class SlackAppServer:
+class SlackAppDevelopmentServer:
     def __init__(
         self, port: int, path: str, app: App, oauth_flow: Optional[OAuthFlow] = None,
     ):
+        """Slack App Development Server
+
+        This is a thin wrapper of http.server.HTTPServer and is good enough
+        for your local development or prototyping.
+
+        However, as mentioned in Python official documents, using http.server module in production
+        is not recommended. Please consider using an adapter (refer to slack_bolt.adapter.*)
+        along with a production-grade server when running the app for end users.
+        https://docs.python.org/3/library/http.server.html#http.server.HTTPServer
+        """
```

Comment:
I really like the detail that's explained here rather than writing it as documentation that could be easily overlooked.

Speaking as someone less experienced with Python, does redirecting the developer to `slack_bolt.adapter.*` give them enough information to add support for a production server? Or do we need additional documentation to guide them through adding an adapter for a production-grade server?

Reply:
I think we should have a page (or a section) about the production server in the documents. The section doesn't exist in Bolt for JS documents as it supports only Express.js and it's totally fine to use its server implementation (similarly to AIOHTTP, Sanic in Python). Once we come up with the page, I would like to have a link to the document here.